test/Zapper.ts

```diff
@@ -184,10 +206,41 @@ async function printPositionInfo (r: ContractTestContext, positionId = 0) {
     console.log(positionId + " position has " + leverage + " lvUSD");
 }
 
+async function getUSDTFromEth (
+    r: ContractTestContext,
+    ethAmountForEstimate = bnFromNum(1),
+): Promise<BigNumber> {
+    const uniswapRouter = await getRouter(r);
+    const amountsReturned = await uniswapRouter.getAmountsOut(
+        ethAmountForEstimate,
+        [addressWETH9, addressUSDT]);
+    return amountsReturned[1];
+}
+
+async function getEthFromUSDT (
```

Comment:
in what case do we need to get Eth from USDT? I think only the reverse case is needed

Reply:
It is used when computing the split between collateral and interest because that calculation assumes a pool between eth and arch so the usdt deposit must first be converted to eth